### ISSUE TITLE
Build fails on i586

### ISSUE THREAD

**Issue by luigino** [EddyRivasLab/hmmer#141]
I assume the arch was dropped in 3.2, could you please [confirm](https://github.com/EddyRivasLab/hmmer/files/2085496/hmmer-3.2-i586_log.txt) ?

**Comment by cryptogenomicon** [comment]
Thanks. I intend for it to work fine on i586, so this is a bug. I don't have an i586 to test on, but I have a suspicion about what's going on. Can you do:

```
    make clean
    make CFLAGS="-O3 -msse2"
```

and see if that works? 

HMMER4's vector code dependencies are much better isolated than H3's, and H3 is now using vector configuration code that I developed in H4. I think what's happening is that the SSE_CFLAGS will need to be used in compiling everything, not just in impl_sse. You can see from the output of `./configure` that the `_MM_SET_FLUSH_ZERO_MODE()` macro is correctly detected at configure time, but is failing at compile time.

**Comment by luigino** [comment]
I see [this](https://github.com/EddyRivasLab/hmmer/files/2085983/hmmer-3.2-i586_O3_sse2_log.txt) now.

**Comment by cryptogenomicon** [comment]
ok, oopsie, I think all that happened is we accidentally clobbered the `-pthread` CFLAG by passing in our own `CFLAGS`. It'll also be helpful to see more verbose output from the build, so let's add V=1 to the `make`. Can you try this for me:

```
    make clean
    make V=1 CFLAGS="-O3 -msse2 -pthread"
```
ditto for your PowerPC build over at #120.

**Comment by luigino** [comment]
Both i586 and ppc64be build correctly now.
The only remaining problem, unsure if related, is that easel seems to ignore $CFLAGS (some extra distro-required switches).

**Comment by cryptogenomicon** [comment]
You're right about CFLAGS, both HMMER and Easel configure scripts were erroneously ignoring any CFLAGS provided by the user. I've checked in changes on our develop branch that fix these issues. Thanks to the gcc compile farm folks, I also got access to an i686 (and a ppc64be) to test them on. The fixes will appear in a 3.2.1 release soon, probably in the next week. Thanks for your help!
